maps/tether/tether_things.dm

```diff
@@ -26,7 +26,7 @@
 		)
 
 /datum/turbolift
-	music = list('sound/music/elevator.ogg')  // Woo elevator music!
+	music = list('modular_citadel/sound/music/elevator.ogg')  // Woo elevator music!
```

Comment:
`//CITADEL CHANGE - changes music here to modular_citadel/sound/music/elevator.ogg`